packages/core/src/session.ts

```diff
@@ -296,10 +297,10 @@ class Session {
   /**
    * Return the bookmark received following the last completed {@link Transaction}.
    *
-   * @return {string[]} A reference to a previous transaction.
+   * @return {string | undefined} A reference to a previous transaction if present.
    */
-  lastBookmark(): string[] {
-    return this._lastBookmark.values()
+  lastBookmark(): string | undefined {
+    return this._bookmarkOut
   }
```

Comment:
Why did the interface changed?

Reply:
I saw a recent question from support regarding the Java driver: they wanted to know if they can safely assume that `.lastBookmark` would only ever return one bookmark. With this signature, there is no doubt about it.